core/chains/evm/txmgr/evm_tx_store_test.go

```diff
@@ -1567,7 +1567,7 @@ func TestORM_CreateTransaction(t *testing.T) {
 
 	t.Run("doesn't insert eth_tx if a matching tx already exists for that pipeline_task_run_id", func(t *testing.T) {
 		id := uuid.New()
-		newTx := txmgr.EvmNewTx{
+		newTx := txmgr.EvmTxRequest{
```

Comment:
nit: Hmm, might make sense to do String search in entire code to replace newTx with txRequest?